distribution/docs/src/main/jdocs/content/_configuring/command-console-intro-contents.adoc

```diff
@@ -0,0 +1,7 @@
+:title: ${command-console} Intro
+:type: configuringCommandConsole
+:status: published
+:summary: ${command-console} Intro.
```

Comment:
So far the summaries have only capitalized the first word but here "Intro" is capitalized as well.